site/next.config.js

```diff
@@ -4,7 +4,6 @@ const withFonts = require("next-fonts");
 module.exports = withFonts(
   withImages({
     pageExtensions: ["page.ts", "page.tsx", "api.ts"],
-    webpack5: false,
```

Comment:
Next.js 12 no longer needs this option. I guess we had it because our initial setup was copied from https://github.com/hashintel/hash/tree/main/packages/hash/frontend